docs/src/clj/om_bootstrap/server.clj

```diff
@@ -20,7 +20,7 @@
 (defn clojurescript []
   (if (dev?)
     (list
-     (h/include-js "http://fb.me/react-0.11.1.js")
+     (h/include-js "http://fb.me/react-0.12.2.js")
```

Comment:
It's needed unless you explicitly add the React file to the :preamble in cljsbuild. Making that change now.